docs/configuring.md

```diff
@@ -0,0 +1,180 @@
+<!--
+* Copyright (c) 2017, 2021 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Configuring your system
+
+When you install a Java&trade; runtime environment on your system you must set the **PATH** environment variable so that the operating system can find the Java programs and utilities to run your application. If your application requires a different library or class file that is not on the **PATH**, you must also set the **CLASSPATH** environment variable. How to set these environment variables is covered in many publications about Java, such as [The Java Tutorials: PATH and CLASSPATH](https://docs.oracle.com/javase/tutorial/essential/environment/paths.html).
```

Comment:
I'm objecting to "must". You don't have to set PATH if apps use the full path or start via a script that uses the JAVA_HOME environment variable or other mechanisms, but you can if you want. 

Setting CLASSPATH is also optional and the side affects should be understood before using it. If set globally it affects all invocations of Java, which may not be desirable.